README.md

```diff
@@ -45,13 +45,19 @@ Read the current JupyterLab documentation on [ReadTheDocs](http://jupyterlab.rea
 
 ### Installation
 
-JupyterLab can be installed using `conda` or `pip`. For more detailed instructions, consult the [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
+JupyterLab can be installed using `mamba`, `conda` or `pip`. For more detailed instructions, consult the [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
```

Comment:
It may be helpful to have a link to the mamba docs, especially since it is not as well known as conda or pip.

Reply:
Thanks! although I did not want to put too much emphasis on mamba over the other solutions.

Reply:
You can add links for conda and pip too, of course. I was thinking that you could just make the word "mamba" a link to the mamba docs, perhaps similar for conda and pip.